pedsnetdcc/utils.py

```diff
@@ -1,28 +1,33 @@
 import urllib.parse
 
 
-def make_conn_str(url, search_path=None, password=None):
+def make_conn_str(uri, search_path=None, password=None):
     """Return a libpq-compliant connection string usable with psycopg2.
 
     If `search_path` is supplied, it is incorporated into the connection string
     via the `options` parameter, overriding any existing search_path in the
-    `url`.
+    `uri`.
 
     If `password` is supplied, it is incorporated into the connection string,
-    overriding any existing password in the `url`.
+    overriding any existing password in the `uri`.
 
     Keywords recognized: host, port, dbname, user, password, options.
 
     Other query parameters are passed as is.
 
     See https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING
 
-    :param str url:
-    :param str search_path:
-    :param str password:
-    :rtype: str
-    """
-    components = urllib.parse.urlparse(url)
+    :param str uri:     The base uri to build the conn string from.
+    :param search_path: An optional search path to include.
+    :type search_path:  str or None
```

Comment:
Is this revenge for my attempts at using RST/Sphinx :-)? I think it's a decent idea to hew to some sort of standard, but I don't care which. The [Google style](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) is more readable and only has the mild disadvantage that the data type is not machine-readable. (What I've been using is a variant of the old-school Sphinx/RST standard which allows the type to be inserted between `param` and the variable name.)